keras_hub/src/models/gemma3/gemma3_tokenizer.py

```diff
@@ -77,20 +77,32 @@ class Gemma3Tokenizer(SentencePieceTokenizer):
 
     backbone_cls = Gemma3Backbone
 
-    def __init__(self, proto, **kwargs):
+    def __init__(self, proto, is_vision_model=True, **kwargs):
```

Comment:
is_vision_model name will be confusing since Gemma3 text models also will have the tokens specific to image and with that condition those mdoels will be tagged as vision model.
Instead rename it something like has_vision_tokens or similar.